microsetta_interface/templates/new_results_page.jinja2

```diff
@@ -72,7 +76,38 @@
         $('#taxonomyTable').DataTable(
             {
                 ajax: {
-                    url: state.public_endpoint + "/taxonomy/present/single/" + state.taxonomy + "/" + state.barcode_prefix + state.sample_id
+                    url: state.public_endpoint + "/taxonomy/present/single/" + state.taxonomy + "/" + state.barcode_prefix + state.sample_id,
+                    dataSrc: function(d){
+                        var representatives = {}
+                        var generaSums = {}
+                        //Collapse to genera (Don't show separate rows per species)
+                        for (var i = 0; i < d.data.length; i++){
+                            //Can't just key by genus because genus can be null/undefined.  Argh.
+                            var genus_key = d.data[i].Kingdom + ";" +
```

Comment:
Can this be domain rather than kingdom?

Reply:
No field named Domain is passed down in the table from public api.  Are you saying you want me to rename Kingdom to Domain in the table displayed to the user, or are you saying we need to add domain to the public api route?

Reply:
Okay, let's leave as is then. "Domain" is more correct than "kingdom" but okay for now.